qa/rpc-tests/smartfees.py

```diff
@@ -90,7 +90,8 @@ def split_inputs(from_node, txins, txouts, initial_split = False):
     # If this is the initial split we actually need to sign the transaction
     # Otherwise we just need to insert the property ScriptSig
     if (initial_split) :
-        completetx = from_node.signrawtransaction(rawtx)["hex"]
+        completetx = from_node.signrawtransaction(
+                rawtx, None, None, "ALL|FORKID")["hex"]
```

Comment:
This is the default anyway, and the tests generally don't specify the sighashtype unless a non-default value is being used.

Reply:
I'm happy to add this, can you just confirm you want it?

Reply:
I'll check if the first patch fixes the issue by itself. If not, then perhaps there is an issue with defaults.